pkg/bicep/testdata/template.bicep

```diff
@@ -2,7 +2,7 @@
 @minLength(2)
 @maxLength(20)
 @allowed(['foo','bar'])
-param testString string = "foo"
+param testString string = 'foo'
```

Comment:
Updated this to single quotes because my bicep linter was giving me errors about double quotes